browser/scripts/application.js

```diff
@@ -2178,6 +2198,16 @@ Application.prototype.setupChat = function() {
 
 	this.chatStore = new E2.ChatStore()
 	this.chat = new E2.Chat($('#chat'))
+	$('.chat-users').draggable({
```

Comment:
also we've been able to do things without jQuery UI so far, and I'd like to keep it that way as long as possible...